api/swagger.yml

```diff
@@ -3540,6 +3540,11 @@ paths:
           description: if set to true, follow only the first parent upon reaching a merge commit
           schema:
             type: boolean
+        - in: query
```

Comment:
Not a fan of parameters with negative names. They're mostly okay as commandline flag, less so in an api. 

However here I have no gear alternative to suggest: the default has to remain with merges. If you *do* have a good alternative I'm very open to suggestions here. 

Otherwise let's go with this one! If we keep on having to add flags, we'll anyway need to end up with an expression parser / GraphQL. So I'm willing to wait until we need another flag.

More importantly is why we need an API change if we just want to filter; that appears on the main comment for this review.